addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -373,17 +373,31 @@ def _import_partner(self, company_id, name, phone, email, vat, country_code=Fals
         partner = self.env['res.partner'] \
             .with_company(company_id) \
             ._retrieve_partner(name=name, phone=phone, email=email, vat=vat, domain=domain)
+        country_code = postal_address.get('country_code')
+        country = self.env.ref(f'base.{country_code.lower()}', raise_if_not_found=False) if country_code else None
```

Comment:
Lil tip for you: if instead of returning None in the else case you return `self.['res.country']` (an empty country record), then when you do `.id`, it returns False. This makes lines 391 removable, it is the same for the state :+1:

Reply:
Nice one, thanks @aboo-odoo 🫡